race_manager.go

```diff
@@ -588,6 +589,13 @@ func (rm *RaceManager) BuildCustomRaceFromForm(r *http.Request) (*CurrentRaceCon
 		trackLayout = ""
 	}
 
+	legalTyres, err := url.PathUnescape(strings.Join(r.Form["LegalTyres"], ";"))
+
+	if err != nil {
+		logrus.WithError(err).Errorf("Couldn't escape legal Tyres list, there may be an issue with the name of a tyre: %s", strings.Join(r.Form["LegalTyres"], ";"))
+		legalTyres = strings.Join(r.Form["LegalTyres"], ";")
```

Comment:
you could set this above the logrus and then use it in the logrus? strings.Join(...) wouldn't need to be repeated then?

Reply:
...in fact if you strings.Join'd the legal tyres outside the if, saved them to a variable, you'd only need to do it once